### ISSUE TITLE
[Broken Website] https://ideas.obsproject.com/

### ISSUE THREAD

**Issue by elibarzilay** [darkreader/darkreader#3799]
**Website**
<https://ideas.obsproject.com/>

**How to Reproduce**
Go there.

**Expected behavior**
Being allowed to darken it.

**Actual behavior**
"This site is in global Dark List".

**Screenshots**

**System info:**
 - OS: Win10
 - Browser: Chrome
 - Darkreader Version: 4.9.16
 
**Additional context**
`obsproject.com` is in the global list, but this site is not dark.
I was surprised, and even more after I read #824 .

**Comment by Gusted** [comment]
> I was surprised, and even more after I read #824 .

As you can see you can override this behavior. 
It's a bug that is similair to #147

**Comment by elibarzilay** [comment]
@Gusted, Are you saying that it's something that will get fixed when that PR gets merged, or that there's a way to do it now?  (If the latter then how?  -- I don't want to start tweaking configuration files manually.)

**Comment by Gusted** [comment]
I meant, just override this behavior by clicking on the `site` button
Example:
![](https://i.imgur.com/paF474m.gif)

**Comment by elibarzilay** [comment]
@Gusted, I now think that there must be some bug somewhere:

* I can't turn it on or off, either by clicking the button or using `Alt+Shift+D`.
* I turned on the new design, and clicking the website button as you show (or the key) still does nothing.
* I then used Settings -> Site list -> added `ideas.obsproject.com` (manually typing it in), and now it's dark.
* The site button, however, is *still* doing nothing!  It shows "Enabled for current website", but clicking does not turn it off.  (And it's still working fine for other sites.)
* If I remove it from the list, it goes back to the above behavior: DR is not active, and clicking the button does nothing.

![](https://i.imgur.com/FAvvjaq.gif)

**Comment by Gusted** [comment]
Hi!

Do you have by any chance `obsproject.com`, the main domain already in the site list?

**Comment by elibarzilay** [comment]
No, I don't think so.  (I went over the list manually, since there's no way to search it, AFAICT.)

**Comment by Gusted** [comment]
Maybe the saved sitelist contains a `bug`, could you reset the settings trough `Settings -> Manage settings -> Reset settings`.

**Comment by elibarzilay** [comment]
@Gusted: Apologies for getting to this so late, but I finally tried it and indeed that resolves the problem but there is a bug...

As soon as I switch to "disabled on all websites by default" it failed again in the same way I saw before.  Now that I have the settings exported, I hand-edited them and re-added my sites to the list in `siteList` (still with an empty `siteListEnabled`), and `ideas.obsproject.com` becomes dark.  Openning it and using Alt+Shift+D still has no effect (it's still dark), but when I look at the new settings, it added it to `siteListEnabled`.  So perhaps the bug is that when I don't have DR on by default toggling a website should get added to `siteList` but instead it gets added to `siteListEnabled`?
